MultisigIntegrationTests/SafeTransactionServiceIntegrationTests.swift

```diff
@@ -22,6 +22,34 @@ class SafeTransactionServiceIntegrationTests: XCTestCase {
         semaphore.wait()
     }
 
+    func test_safeInfo_whenSendingNotASafe_returns404Error() {
+        let semaphore = DispatchSemaphore(value: 0)
+        DispatchQueue.global().async {
+            try? XCTAssertThrowsError(try self.service.safeInfo(at: "0xc778417E063141139Fce010982780140Aa0cD5Ab")) {
```

Comment:
Fun fact (seems like Xcode bug):
It should work without 'try?', but the compiler produces an error "Invalid conversion from throwing function of type '() throws -> Void' to non-throwing function type '@convention(block) () -> Void'"
With try? compiler complains "No calls to throwing functions occur within 'try' expression"

Reply:
I think it is because the async {} method expects non-throwing closure. Might be that the XCTAssertThrows actually rethrows?